### ISSUE TITLE
How do I disable the maximize button?

### ISSUE THREAD

**Issue by vs-123** [Boscop/web-view#249]
### How do I disable the maximize button?

I tried setting .resizable(false) but it wont disable the maximize button.

My code:

```rust
use web_view::*;

fn main() {
    let html_content = "<html><body><h1>Saluton Mundo</h1></body></html>";

    web_view::builder()
        .title("My Project")
        .content(Content::Html(html_content.as_str()))
        .size(320, 480)
        .resizable(false)
        .debug(true)
        .user_data(())
        .invoke_handler(|_webview, _arg| {
            println!("{:?}", _webview);
            Ok(())
        })
        .run()
        .unwrap();
}
```

**Comment by 3442853561** [comment]
Don't worry. I think that the designer forgot to maximize the window is also part of changing the size. and I tried to fix it

**Comment by zxey** [comment]
As of 1bdbd4a93c542e150e918ae961dbcab03a3ea0db and 152c436558e9ef93d9ae892b65769880454ba726 should be fixed, thanks @3442853561.
